src/components/ConfigureImageClassification/index.js

```diff
@@ -44,6 +44,15 @@ export default ({ iface, onChange }) => {
       variant="flat"
       defaultAnswers={defaultAnswers}
       onQuestionChange={(questionId, newValue, answers) => {
+        if (Array.isArray(newValue))
+          newValue = newValue.filter((json, index, array) => {
+            for (var i = 0; i < index; i++) {
+              if (array[i].id === json.id) {
+                return false
+              }
+            }
+            return true
+          })
```

Comment:
This is inefficient (`n^2`) and a bit verbose, try something like this:

```javascript
if (Array.isArray) {
  newValue = Array.from(new Set(newValue))
}
```

Reply:
Set doen't do exactly what i wish but i do have cleaning up a bit by erasing the loop

Reply:
Here's an `O(n)` implenentation

```javascript
if (Array.isArray(newValue)) {
  const idMap = {}
  for (const item of newValue) {
    if (idMap[item.id]) continue
    idMap[item.id] = item
  }
  newValue = Object.values(idMap)
}
```